src/composables/useValidation.ts

```diff
@@ -14,6 +14,30 @@ export const useValidation = () => {
     return regex.test(value);
   };
 
+  const isPhone = (value: string): boolean => {
+    // 6-18 characters (extreme length with delimiters)
+    if (value.length < 6) {
+      return false;
+    }
+    if (value.length > 18) {
+      return false;
+    }
+    /**
+     * Match international phone number entered with delimiters
+     * (spaces, dots, brackets, etc.)
+     *
+     * Note: does not account for specific mistakes such as:
+     * Leading Zero in Area Code: +420 012 345 678
+     * Too Many Digits: +420 123 456 7890
+     * Starts with an Invalid Country Code: +999 123 456 789
+     *
+     * https://uibakery.io/regex-library/email
+     */
+    const regex =
```

Comment:
What we need is more exact catch phone number format which respects our internal rules (catch exact +420xxxxxxxxx without space) and the rules that apply in the [Czechia for the format of telephone numbers](https://en.wikipedia.org/wiki/Telephone_numbers_in_the_Czech_Republic) . Did you really test regex (regex catch number 00 00 00) ?